src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1776,6 +1776,18 @@ private static EmitResult ToEmitResultAndFree(DiagnosticBag diagnostics, bool su
             bool emitPortablePdb = moduleBeingBuilt.EmitOptions.DebugInformationFormat == DebugInformationFormat.PortablePdb;
             string pdbPath = (pdbStreamProvider != null) ? (moduleBeingBuilt.EmitOptions.PdbFilePath ?? FileNameUtilities.ChangeExtension(SourceModule.Name, "pdb")) : null;
 
+            // The PDB path is emitted in it's entirety into the PE.  This makes it impossible to have deterministic
```

Comment:
I don't think this can be done in the command line compiler.  There are two PDB paths we have to consider:

1. The path where the PDB is emitted on disk.  
2. The path embedded in the PE.

If we did the work in the compiler it would effect both options.  Doing the work here allows us to properly separate the concerns.